core/trino-main/src/test/java/io/trino/operator/exchange/TestLocalExchange.java

```diff
@@ -49,7 +49,9 @@
 import java.util.Optional;
```

Comment:
`testScaleWriters` in BaseHiveConnectorTest should be updated to disable task writer scaling so that it can test the existing writer scaling in isolation.
We should also have a test similar to that specifically for task writer scaling.

Reply:
Added new test `testScaleTaskWriters`